.github/workflows/silabs-firmware-build.yaml

```diff
@@ -64,6 +64,16 @@ jobs:
               --export-destination=${{ inputs.firmware_name }} \
               --copy-proj-sources --copy-sdk-sources --new-project --force \
               --configuration="${{ inputs.configuration }}"
+
+      - name: Set Baudrate
+        env:
+          baudrate: ${{ fromJSON(inputs.metadata_extra).baudrate }}
+        if: ${{ env.baudrate != 115200 }}
```

Comment:
Hm, what if their default changes? I mean this is essentially a nop, I think I'd rather suggest to avoid that risk and drop this if... Thoughts?